modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -4065,6 +4064,12 @@ export const console: ConsoleNS = {
                                 label: "Authentication context class",
                                 placeholder: "Enter authentication context class"
                             },
+                            customAuthenticationContextClass: {
+                                ariaLabel: "Custom Authentication context class",
+                                hint: "Specify the custom authentication context class",
+                                label: "Custom authentication context class",
+                                placeholder: "Enter custom authentication context class"
+                            },
```

Comment:
translation should be added to other languages as well. Check french and sinhala

Reply:
Translations had already been added for the English language only. Adding the french and sinhala translations for the whole file would be out of scope of this PR.